### ISSUE TITLE
Compilation breaks on kernel version 5.15.84

### ISSUE THREAD

**Issue by wikstorm** [kbarni/LCCV#15]
After running `rpi-update`, installing dependencies for LCCV and compiling it fails with this:

```
In file included from /home/pi/LCCV/include/lccv.hpp:9,
                 from /home/pi/LCCV/src/lccv.cpp:1:
/home/pi/LCCV/include/libcamera_app.hpp: In constructor ‘FrameInfo::FrameInfo(libcamera::ControlList&)’:
/home/pi/LCCV/include/libcamera_app.hpp:194:20: error: invalid type argument of unary ‘*’ (have ‘int’)
  194 |    exposure_time = *exp;
      |                    ^~~~
compilation terminated due to -Wfatal-errors.
make[2]: *** [CMakeFiles/liblccv.dir/build.make:82: CMakeFiles/liblccv.dir/src/lccv.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:95: CMakeFiles/liblccv.dir/all] Error 2
make: *** [Makefile:171: all] Error 2
```

After reading old issues it seemed like this also was an issue back when libcamera had an update, but this was seemingly fixed. It seems odd then, after keeping my system up to date, that this appears again.

Kernel version: 5.15.84-v7l+
GCC version: 10.2.1
CMake version: 3.18.4
libcamera version (from `apt show libcamera-dev`): 0~git20221130+0684c373-1

Any idea what causes this? Should I reinstall Bullseye?

**Comment by kbarni** [comment]
*libcamera* gets updated often, it is still in heavy development. The API is not fixed, so updates can break software that's based upon this library, this already happened at least 2 times during 2022.

I'll look into this error, but I won't have time before mid January...

**Comment by wikstorm** [comment]
Yes, although it feels like breaks like these should be visible on the libcamera-apps repository. No issue made, no recent update to those lines of code, nothing. I will flash the rpi and start over, see if that makes any difference.

**Comment by JuergenSmo** [comment]
@kbarni : Hello, after a lot of changes I was able to compile it. But now I get a segemtation fault when I call 'getVideoFrame'. Is there a possibility that you take a look on what I have? I can send it to you as, but I do not want to upload anything as it does not work.

**Comment by kbarni** [comment]
Thanks @JuergenSmo for your work!
Yes, you can send it so I look at it.
Otherwise create a fork, put your version on that fork, and make a pull request when the fork is working (that's more *github* way...)

**Comment by JuergenSmo** [comment]
I created a new fork you can find it at https://github.com/JuergenSmo/LCCV.
Many thanks @kbarni !
